### PR TITLE
Customized date groups

### PR DESCRIPTION
**Pull request by sikandartariq1**

### CODE REVIEW

File: dev/App.vue [hunk 1]
```diff
@@ -91,6 +91,8 @@
           :date-one="inlineDateOne"
           :months-to-show="2"
           :disabled-dates="['2018-04-30', '2018-05-10', '2018-12-14']"
+          :customized-dates="[['2019-03-21', '2019-03-22', '2019-03-23'], ['2019-03-21', '2019-03-22', '2019-03-23']]"
+          :customized-groups="['booked', 'not-available']"
```
Comment (sikandartariq1) [inline dev/App.vue]: @MikaelEdebro not sure if this is the best approach (using two arrays for one purpose)... Hash maybe?
`customized-dates="['booked': ['2019-03-21', '2019-03-22', '2019-03-23'], 'not-available': ['2019-03-21', '2019-03-22', '2019-03-23']]`

**Comment by MikaelEdebro** [comment]
I think it's best to use:

```html
:customized-dates="[
  {
    dates: ['2019-03-21', '2019-03-22', '2019-03-23'],
    cssClass: 'booked'
  },
  {
    dates: ['2019-03-29', '2019-03-30'],
    cssClass: 'not-available'
  },
]"
```

That way we only need one property, and it's really clear what class to add to the specified dates.

**Comment by sikandartariq1** [comment]
perfect!

**Comment by sikandartariq1** [comment]
Updated!! Please take a look @MikaelEdebro

**Comment by MikaelEdebro** [comment]
Hi! Sorry for the delay. Have been really busy with work and other stuff. :)

I'll have a look at this in the upcoming days. I'll just have to pull the branch and test locally, and write some docs in the readme.

**Comment by sikandartariq1** [comment]
No problem! Let me know if I can help with something

**Comment by MikaelEdebro** [comment]
@sikandartariq1 It would be great if you can document this in https://github.com/MikaelEdebro/vue-airbnb-style-datepicker/blob/master/docs/INSTALLATION.md#props-and-events

**Comment by MikaelEdebro** [comment]
Thanks for the contribution. This is now released in https://github.com/MikaelEdebro/vue-airbnb-style-datepicker/releases/tag/v2.6.0

**Comment by sikandartariq1** [comment]
Cool, thanks!

**Pull request merged**